### PR TITLE
Make the logging page look more like the dashboard page

### PR DESCRIPTION
**Pull request by rmars**
`parent: rmars/style-followup`

Follow up to #499 and #497

I've added our own js/css by creating a filter that adds things to the response body. Let me know your opinions on this.

Also colors/styles can be tweaked, opinions welcome.

screen shot 2016-06-22 at 4 33 31 pm

### CODE REVIEW

File: admin/src/main/scala/io/buoyant/admin/StyleOverrideFilter.scala [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  val stylesAndJavascripts =
+    s"""
+    <link type="text/css" href="/files/css/logger-overrides.css" rel="stylesheet"/>
+    <link type="text/css" href="/files/css/dashboard.css" rel="stylesheet"/>
+    <link type="text/css" href="/files/css/dashboard-shared.css" rel="stylesheet"/>
+    <script src="/files/js/logger-overrides.js"></script>
+      """
+
+  def apply(req: Request, svc: Service[Request, Response]) = {
+    svc(req).map(response => {
```
Comment (adleong) [inline admin/src/main/scala/io/buoyant/admin/StyleOverrideFilter.scala]: style nit:

```
svc(req).map { response =>
  ...
}

**Comment by adleong** [comment]
:star: 🎄

File: admin/src/main/resources/io/buoyant/admin/css/logger-overrides.css [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+.table-striped>tbody>tr:nth-child(odd)>td, .table-striped>tbody>tr:nth-child(odd)>th {
+  background-color: #161963;
+}
+
+.table-striped>tbody>tr>td {
+  border-top: 2px solid gray;
+}
+
+.table-striped>thead>tr {
+  border-bottom: 3px solid gray;
```
Comment (esbie) [inline admin/src/main/resources/io/buoyant/admin/css/logger-overrides.css]: my preference for these borders is `1px solid dimgray` or `1px solid slategray`

File: admin/src/main/resources/io/buoyant/admin/js/logger-overrides.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  '        </div>' +
+  '        <div id="navbar" class="collapse navbar-collapse">' +
+  '          <ul class="nav navbar-nav">' +
+  '            <li><a href="/delegator">dtab</a></li>' +
+  '            <li><a href="/admin/logging">logging</a></li>' +
+  '            <li><a href="https://linkerd.io/help/">help</a></li>' +
+  '          </ul>' +
+  '        </div>' +
+  '      </div>' +
+  '    </nav>"';
```
Comment (esbie) [inline admin/src/main/resources/io/buoyant/admin/js/logger-overrides.js]: Can we send this html down as a string inside a script tag? Like:
```
<script id="linkerd-navbar" type="application/json">{"navbarToInject": "<nav class=..."}</script>
```

This way ideally we can reuse the nav bar from other pages without duplication.

**Comment by rmars** [comment]
(addressing esbie's comments in #505)

**Pull request merged**